donut/modules/courses/templates/scheduler.html

```diff
@@ -461,7 +486,58 @@ <h1>Course Scheduler</h1>
     function displayUnits() {
       units.text('Units: ' + String(coursesUnits))
     }
-
+  $('#modal').on('show.bs.modal', function(e) {
+    var courseNum = $(e.relatedTarget).data("courseNum");
+    var courseId = $(e.relatedTarget).data("courseId");
+    displayNotes(courseNum, courseId); 
+  });
+  $('#editButton').click(function(e) {
+    $('#postNotes').show();
+    $('#viewNotes').hide();
+    $('#editButton').hide();
+  });
+  $('#postNotes').submit(function(e) {
+    e.preventDefault();
+    $.ajax({
+      url: '/1/scheduler/edit_notes',
```

Comment:
Use `url_for()`?

Reply:
I was thinking to keep it consistent with the old code which does not use url_for